pyk/src/pyk/rewrite.py

```diff
@@ -103,3 +160,40 @@ def combine_dicts(dict1: Optional[Mapping[K, V]], dict2: Optional[Mapping[K, V]]
 def combine_all(*dicts: Optional[Mapping[K, V]]) -> Optional[Dict[K, V]]:
     unit: Optional[Dict[K, V]] = {}
     return reduce(combine_dicts, dicts, unit)
+
+
+def rewrite(rule: Tuple[KInner, KInner], term: KInner) -> KInner:
+    """
+    Rewrite a given term at the top with the supplied rule.
+
+    :param rule: A rewrite rule of the form (lhs, rhs).
+    :param term: Term to rewrite.
+    :return: The term with the rewrite applied once at the top.
+    """
```

Comment:
Are these docstrings helpful to keep anymore? Will they be used eventually maybe?

Reply:
> Will they be used eventually maybe?

I think they should be (though I have to admit, I really have to force myself to write them).